xclim/indices/run_length.py

```diff
@@ -786,7 +889,7 @@ def season(
     Returns
     -------
     xr.Dataset
-        "dim" is reduced to "season_bnds" with 2 elements : season start and season end, both indices of da[dim].
+        A Dataset with three variables : start, end and length of the season.
```

Comment:
Live by the Oxford comma, die by the Oxford comma.
```suggestion
        A Dataset with three variables : start, end, and length of the season.
```